examples/client/Locomotion/src/pages/Payments/credit-cards.tsx

```diff
@@ -31,21 +61,69 @@ export default ({
   return (
     <CardsListContainer>
       <View>
-        {usePayments.paymentMethods.map((paymentMethod : any) => (
-          <PaymentMethodsContainer>
-            <CreditCardsContainer>
-              <PaymentMethod {...paymentMethod} onPress={() => navigate(MAIN_ROUTES.CARD_DETAILS, { paymentMethod })} />
-            </CreditCardsContainer>
-          </PaymentMethodsContainer>
-        ))}
+        <PaymentMethodsContainer>
+          {defaultMethod && defaultMethod.id !== cashPaymentMethod.id
```

Comment:
defaultMethod?.id

Reply:
types error, cant put nullable on it cause it has one more field than interface